.babelrc

```diff
@@ -1,3 +1,22 @@
 {
-  "presets": ["es2015", "stage-2"]
-}
\ No newline at end of file
+  "env": {
+    "build": {
+      "presets": [
+        [
+          "es2015",
```

Comment:
Wonder if we should be using `babel-preset-env` now?

Reply:
Yes, we could switch this out if we liked. I think I may have attempted to use it at the time as well but it caused some odd builds which bugged out during testing.

However a lot has changed since this was originally written and we should make the change. I would like to add that I wish we didn't have to support `IE11`. 😹